btcjson/walletsvrresults.go

```diff
@@ -48,11 +48,13 @@ type InfoWalletResult struct {
 	Proxy           string  `json:"proxy"`
 	Difficulty      float64 `json:"difficulty"`
 	TestNet         bool    `json:"testnet"`
+	MoneySupply     float64 `json:"moneysupply"`
 	KeypoolOldest   int64   `json:"keypoololdest"`
 	KeypoolSize     int32   `json:"keypoolsize"`
 	UnlockedUntil   int64   `json:"unlocked_until"`
 	PaytxFee        float64 `json:"paytxfee"`
 	RelayFee        float64 `json:"relayfee"`
+	StakingStatus   string  `json:"staking status"`
```

Comment:
This JSON tag doesn't look right... Also, is this a field returned by `btcd`?

Reply:
this is in the output of the getinfo command ... and it looks just like that ... and it works

![example](https://user-images.githubusercontent.com/20052716/73671109-6d038f80-46bb-11ea-8b23-483a3178052e.jpg)

Reply:
I'd ran this as well with no issuese

Reply:
Is "staking status" something `btcd` reports? I can't find it in the code. If this is something used by a non-Bitcoin chain I don't think `btcd` is the correct place to implement this.

Reply:
there are several thousand cryptocurrencies and, accordingly, *coind ... i do not think it is right for each cryptocurrency to write a separate rpcClient ...

Reply:
I think that is absolutely right, that each crypto currency should maintain their own RPC client. If we added all possible fields to the `btcd/rpcclient` struct, that would make users of the Bitcoin client (which is what this package is made for) super confused. 

I'm strongly opposed to adding any altcoin functionality to this project. Forking it and maintaining a 1-commit difference is not hard.

Reply:
You have the right to your opinion, I have the right to my opinion ... I think that we do not need 10,000 cats that differ in the shape of the spots on the nose ...